src/Resources/templates/htmlTableOutput.phtml

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>SensioLabs DeprecationDetector</title>
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
```

Comment:
This is how the "proof of concept" looks:

![deprecator_styles](https://cloud.githubusercontent.com/assets/73419/10667878/00f00098-78da-11e5-9e74-9d7c13285b3c.png)

Rows show a highlight color when moving hte mouse over them:

![deprecator-styling](https://cloud.githubusercontent.com/assets/73419/10667884/03482d98-78da-11e5-966c-137bebbd65f9.gif)

Is this concept something we can iterate over until we are satisfied ... or should we abandon this idea?